habitat-lab/habitat/tasks/rearrange/rearrange_grasp_manager.py

```diff
@@ -213,15 +217,16 @@ def snap_to_marker(self, marker_name: str) -> None:
         if self._kinematic_mode:
             return
 
-        self._snap_constraints = [
-            self.create_hold_constraint(
-                RigidConstraintType.PointToPoint,
-                mn.Vector3(0.0, 0.0, 0.0),
-                mn.Vector3(*marker.offset_position),
-                marker.ao_parent.object_id,
-                marker.link_id,
-            ),
-        ]
+        if self._automatically_update_snapped_object:
+            self._snap_constraints = [
```

Comment:
Curious what is the use-case here?

Reply:
This is for dynamic simulations. The other code path is for kinematic simulations.